private/bufpkg/bufimage/bufimageutil/image_filter.go

```diff
@@ -436,6 +436,10 @@ func (b *sourcePathsBuilder) remapOneof(
 	sourcePath protoreflect.SourcePath,
 	oneof *descriptorpb.OneofDescriptorProto,
 ) (*descriptorpb.OneofDescriptorProto, bool, error) {
+	if mode, ok := b.closure.elements[oneof]; ok && mode == inclusionModeExcluded {
+		// Oneofs can be explicitly excluded when all fields for it are excluded.
```

Comment:
I find this sentence confusing because, in such cases, the oneof is _implicitly_ excluded (I think explicitly excluding it would mean referring to it directly by name from an `exclude_types` filter).